subeditor.js.org

Shirtiny opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

Shirtiny commented:
> 
> 
> CNAME'ing to a directory on a custom domain doesn't work. Did you want to CNAME to `shirtiny.github.io/SubEditor` instead?

Yes,I see.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```subeditor.js.org``` to make it work.
I'll merge after we have "contact".

Shirtiny commented:
> 
> 
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `subeditor.js.org` to make it work.
> I'll merge after we have "contact".

I've already added the `CNAME`.  It works, thank you : ) .

indus commented:
Welcome https://subeditor.js.org
